clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1061,11 +1061,13 @@ def Most : DiagGroup<"most", [
  ]>;
 
 // Thread Safety warnings
-def ThreadSafetyAttributes : DiagGroup<"thread-safety-attributes">;
-def ThreadSafetyAnalysis   : DiagGroup<"thread-safety-analysis">;
-def ThreadSafetyPrecise    : DiagGroup<"thread-safety-precise">;
-def ThreadSafetyReference  : DiagGroup<"thread-safety-reference">;
-def ThreadSafetyNegative   : DiagGroup<"thread-safety-negative">;
+def ThreadSafetyAttributes       : DiagGroup<"thread-safety-attributes">;
+def ThreadSafetyAnalysis         : DiagGroup<"thread-safety-analysis">;
+def ThreadSafetyPrecise          : DiagGroup<"thread-safety-precise">;
```

Comment:
I would suggest to undo the indentation changes here, as the flag is intended to be temporary.

Reply:
Done.